examples/client_simple/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/teeworlds-go/protocol/protocol7"
+	"github.com/teeworlds-go/protocol/teeworlds7"
+)
+
+func main() {
+	ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt, syscall.SIGTERM)
+	defer cancel()
+
+	client := teeworlds7.NewClient()
+	client.Name = "nameless tee"
+
+	client.OnPacket(func(packet *protocol7.Packet) bool {
+		fmt.Printf("received packet with %d message(s)\n", len(packet.Messages))
```

Comment:
This is still printing every message in the "client_simple" example. Seems quite redundant to the "client_verbose" maybe you could name it differently to describe what this example is about. Or merge them together. Or make it not spam print.

Reply:
changed code a little bit to actually fit some specific use case and renamed the example application to "client_echo".